akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/whiteboard/WhiteboardApp2x.scala

```diff
@@ -1,73 +1,76 @@
-  with GetWhiteboardAnnotationsReqMsgHdlr {
-
-  val log = Logging(context.system, getClass)
-
-  def sendWhiteboardAnnotation(annotation: AnnotationVO, drawEndOnly: Boolean, liveMeeting: LiveMeeting): AnnotationVO = {
-    //    println("Received whiteboard annotation. status=[" + status + "], annotationType=[" + annotationType + "]")
-    var rtnAnnotation: AnnotationVO = annotation
-
-    if (WhiteboardKeyUtil.DRAW_START_STATUS == annotation.status) {
-      rtnAnnotation = liveMeeting.wbModel.addAnnotation(annotation.wbId, annotation.userId, annotation)
-    } else if (WhiteboardKeyUtil.DRAW_UPDATE_STATUS == annotation.status) {
-      if (WhiteboardKeyUtil.PENCIL_TYPE == annotation.annotationType) {
-        rtnAnnotation = liveMeeting.wbModel.updateAnnotationPencil(annotation.wbId, annotation.userId, annotation)
-      } else {
-        rtnAnnotation = liveMeeting.wbModel.updateAnnotation(annotation.wbId, annotation.userId, annotation)
-      }
-    } else if (WhiteboardKeyUtil.DRAW_END_STATUS == annotation.status) {
-      if (WhiteboardKeyUtil.PENCIL_TYPE == annotation.annotationType) {
-        rtnAnnotation = liveMeeting.wbModel.endAnnotationPencil(annotation.wbId, annotation.userId, annotation, drawEndOnly)
-      } else {
-        rtnAnnotation = liveMeeting.wbModel.updateAnnotation(annotation.wbId, annotation.userId, annotation)
-      }
-    } else {
-      //	    println("Received UNKNOWN whiteboard annotation!!!!. status=[" + status + "], annotationType=[" + annotationType + "]")
-    }
-
-    rtnAnnotation
-  }
-
-  def getWhiteboardAnnotations(whiteboardId: String, liveMeeting: LiveMeeting): Array[AnnotationVO] = {
-    //println("WB: Received page history [" + msg.whiteboardId + "]")
-    liveMeeting.wbModel.getHistory(whiteboardId)
-  }
-
-  def clearWhiteboard(whiteboardId: String, requesterId: String, liveMeeting: LiveMeeting): Option[Boolean] = {
-    liveMeeting.wbModel.clearWhiteboard(whiteboardId, requesterId)
-  }
-
-  def undoWhiteboard(whiteboardId: String, requesterId: String, liveMeeting: LiveMeeting): Option[AnnotationVO] = {
-    liveMeeting.wbModel.undoWhiteboard(whiteboardId, requesterId)
-  }
-
-  def getWhiteboardAccess(whiteboardId: String, liveMeeting: LiveMeeting): Boolean = {
-    liveMeeting.wbModel.getWhiteboardAccess(whiteboardId)
-  }
-
-  def modifyWhiteboardAccess(whiteboardId: String, multiUser: Boolean, liveMeeting: LiveMeeting) {
-    liveMeeting.wbModel.modifyWhiteboardAccess(whiteboardId, multiUser)
-  }
-
-  def filterWhiteboardMessage(whiteboardId: String, liveMeeting: LiveMeeting): Boolean = {
-    // Need to check if the wb mode change from multi-user to single-user. Give 5sec allowance to
-    // allow delayed messages to be handled as clients may have been sending messages while the wb
-    // mode was changed. (ralam nov 22, 2017)
-    if (!liveMeeting.wbModel.getWhiteboardAccess(whiteboardId) && liveMeeting.wbModel.getChangedModeOn(whiteboardId) > 5000) true else false
-  }
-}

```

Comment:
I didn't change that many, just adding two lines.

Reply:
did you maybe change the file encoding or added/removed Byte Order Mark?

Reply:
I don't think so. I've only fixed the conflict and github seems to have behaved wrongly..

Reply:
@basisbit It is likely that I have changed the line-feed code. But I couldn't manage to revert completely. After copied from the dev branch it looks better now, highlighting the modified lines by a dark green, so I leave it as it is now. Let me know if it gives any inconvenience.